torch_xla/csrc/runtime/pjrt_computation_client.cc

```diff
@@ -677,8 +677,9 @@ PjRtComputationClient::ExecuteReplicated(
   TF_VLOG(5) << "ExecuteReplicated acquiring PJRT device lock for "
              << spmd_device_str << " Done";
 
-  std::optional<std::vector<xla::PjRtFuture<xla::Status>>> returned_futures(
-      devices.size());
+  std::optional<std::vector<xla::PjRtFuture<xla::Status>>> returned_futures =
```

Comment:
Is this change from the Alibaba PR?

Reply:
yes. I removed it since the Alibaba's PR has been merged